daemon/containerd/image_exporter.go

```diff
@@ -123,8 +136,13 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, outSt
 		}
 
 		name := img.Name
-		if named, err := reference.ParseNormalizedNamed(img.Name); err == nil {
-			name = reference.FamiliarName(named)
+		loadedMsg := "Loaded image"
+
+		if isDanglingImage(img) {
+			name = img.Target.Digest.String()
+			loadedMsg = "Loaded image ID"
```

Comment:
The conditional message is what the non-c8d Docker does.
Wondering if it's crucial to keep the output compatible? Maybe we could just have one "Loaded image: <image>" for both cases?

Reply:
It looks like there's at least some code checking for this; https://grep.app/search?q=Loaded%20image%20ID

I agree that we should look into some of these; perhaps it should show _more_ details (if multiple architectures are loaded etc etc), and close with a `Loaded image` (or something).

if we're gonna "break" output, we should probably do some of that with a big bang; related to that;

- https://github.com/moby/moby/issues/41838